pkg/tui/go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/unowned-ai/recall/pkg/tui
 
-go 1.24.2
+go 1.23
```

Comment:
I really don't think this needs modification. Please revert this change on your branch.